scio-test/src/test/scala/com/spotify/scio/testing/JobTestTest.scala

```diff
@@ -98,6 +98,16 @@ object BigQueryJob {
   }
 }
 
+object BigQueryJobReadingTheSameTableWthDifferentPredicateAndProjections {
```

Comment:
Ugly long name. Should probably be moved to a separate test suite

Reply:
Yeah can we come up with a shorter one?

Reply:
I'll try :D